js/varien/js.js

```diff
@@ -33,122 +33,33 @@ function setPLocation(url, setFocus){
  *
  * @param elements - array of elements to be decorated
  * [@param decorateParams] - array of classes to be set. If omitted, all available will be used
+ * @deprecated
```

Comment:
Is it better to not remove the code first, only mark it deprecated?

Reply:
I would deprecate it in main and remove it in next, otherwise it will take forever and we already have many breaking changes in next.
I could open a PR with deprecating this funcions in main but I wouldn't deprecate them in next, those old things from the past have to go as fast as possible IMHO